src/viz/index.ts

```diff
@@ -40,23 +42,11 @@ export const createFromTemplate = async (answers: Answers): Promise<number> => {
     }
   });
 
-  await util.spinnify('Building & pushing dev deployment...', async () => {
-    if (answers.yarn) {
-      await util.exec('yarn run build:dev', {cwd: projectPath}, false);
-      await util.exec('yarn run push:dev', {cwd: projectPath}, false);
-    } else {
-      await util.exec('npm run build:dev', {cwd: projectPath}, false);
-      await util.exec('npm run push:dev', {cwd: projectPath}, false);
-    }
-  });
-
   console.log(
-    `\
-cd ${projectName} to start working on your viz!\n\
-\n\
-Your viz has been deployed to: ${devBucket}\n\
+    `
+Created new community viz: ${projectName}
 \n\
-For next steps, see ${projectName}/README.md\
+${chalk.black.bgCyan.bold('cd ' + projectName)} and ${chalk.black.bgCyan.bold('npm run start')} to begin working on your viz!\n\
```

Comment:
Could you pull out the chalk commands into variables so the string literal is easier to follow?